docs/tools/markdown-it-rule-example.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
```

Comment:
Can you add some comments in this file explaining how this works?

Reply:
I've added some comments in https://github.com/eslint/eslint/pull/17718/commits/b5b247a7de3b46c56b1cbd02a7371ce9281ab789.